freqtrade/freqai/freqai_interface.py

```diff
@@ -446,11 +451,6 @@ def data_cleaning_train(self, dk: FreqaiDataKitchen) -> None:
             dk.use_DBSCAN_to_remove_outliers(predict=False, eps=eps)
             self.dd.old_DBSCAN_eps[dk.pair] = dk.data['DBSCAN_eps']
 
-        if ft_params.get('inlier_metric_window', 0):
```

Comment:
i'm ... not sure i understand how moving this from "here" to the top of this function will change if you can combine it or not ?

Reply:
`compute_inlier_metric‘ adds a feature to the feature set.